.clang-format

```diff
@@ -10,7 +11,67 @@ AlwaysBreakTemplateDeclarations: true
 AlwaysBreakAfterReturnType: All
 PointerAlignment: Left
 AllowShortIfStatementsOnASingleLine: false
-BreakBeforeBraces: Allman
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass:      true
+  AfterControlStatement: Always
+  AfterEnum:       true
+  AfterFunction:   true
+  AfterNamespace:  true
+  AfterObjCDeclaration: true
+  AfterStruct:     true
+  AfterUnion:      false
+  AfterExternBlock: true
+  BeforeCatch:     true
+  BeforeElse:      true
+  BeforeLambdaBody: false
+  IndentBraces:    true
+  BeforeWhile: false
+  IndentBraces: false
+  SplitEmptyFunction: true
+  SplitEmptyRecord: true
+  SplitEmptyNamespace: true
+
+# Disable formatting options which may break tests.
+SortIncludes: false
+ReflowComments: false
+
+# Indent preprocessor directives
+IndentPPDirectives: AfterHash
```

Comment:
check this vs existing clang-format-6.0 and existing repo

Reply:
This setting is unchanged from the previous style file, and is in agreement with the code in the repo.